File: file.py
import ConfigParser
import StringIO
import sys

iniPath='config.ini'
expectedVariables = ['host', 'port', 'user', 'pass']
config = {}

try:
    iniAsStr = '[section]\n' + open(iniPath, 'r').read()
    iniFp = StringIO.StringIO(iniAsStr)
    configRead = ConfigParser.RawConfigParser()
    configRead.readfp(iniFp)
except:
    print "Missing config file: " + iniPath
    sys.exit()

for var in expectedVariables:
    if var not in configRead.options("section"):
        print "Missing config variable: " + var
	sys.exit()
    config[var] = configRead.get("section", var).strip('"')


print config['host']
print config['port']
print config['user']
print config['pass']

File: file.php
<?php
$iniPath = 'config.ini';
$expectedVariables = array('host', 'port', 'user', 'pass');

if (!is_readable($iniPath)) {
    die("Missing config file: $iniPath\n");
}
$config = parse_ini_file($iniPath);
if (!$config) {
    die("Empty or invalid config file \n");
}
foreach ($expectedVariables as $var) {
    if (!isset($config[$var])) {
        die("Missing config variable: $var\n");
    }
}
echo $config['user'] . "\n";
echo $config['host'] . "\n";
echo $config['pass'] . "\n";
echo $config['port'] . "\n";
